### ISSUE TITLE
Wrong experiment associated with Study_277

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDB#145]
@ftzohra22 do you know which study these Crohn's Disease experiments/signatures are supposed to belong to? Study https://bugsigdb.org/Study_277 is about Cystic Fibrosis. This may require help from @tosfos to correct once we find the right study.

**Comment by lwaldron** [comment]
By the way @ftzohra22, from the related #143 the likely cause is:
> It looks like the Study and Experiment were created together. Then someone deleted the Study (see [here](https://bugsigdb.org/w/index.php?title=Special:Log&page=Study+567)) without deleting the Experiment page, which is very easy to do. When a new Study page was then created, the Experiment 1 page was still in the wiki, which caused all the confusion. 

Also, would you take a look at https://bugsigdb.org/w/index.php?title=Help:Cleanup&oldid=99217 for any other orphaned pages that may need to be cleaned up?

**Comment by ftzohra22** [comment]
I'll take a look at both of these and get back to you

**Comment by ftzohra22** [comment]
@lwaldron for https://bugsigdb.org/Study_277 it doesn't seem like a bug on the wiki's end. I went back to our google spreadsheet and it looks like there's an incorrect PMID associated with that entry on the spreadsheet which was carried on with the import. However, I was able to find the correct PMID by searching some keywords associated with the study in pubmed. Fixed and reviewed this paper in bugsigdb. Thank you for catching this. I'll keep you posted if I find any other cases like this.
